.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Report a bug in knative/client
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+<!-- If you need to report a security issue with Knative, send an email to knative-security@googlegroups.com. -->
+## In what area(s)?
+
+<!-- Remove the '> ' to select -->
+
+<!--
+Classifications:
+> /kind good-first-issue
+> /kind doc
+> /kind cleanup
+-->
+
+## What version of Knative Client?
+
+<!-- Delete all but your choice -->
+
+> nightly
+> 0.1.x
+> 0.2.x
+> Output of git describe --dirty
+
+## What version of Knative Serving?
+
+<!-- Delete all but your choice -->
+
+> nightly
+> 0.5.x
+> 0.6.x
+> 0.7.x
+> 0.8.x
+> Output of git describe --dirty
```

Comment:
We are introducng knative serving as client dependency in the vendor dir, so there is a direct relationship between the Kn client version and the serving used as dependency.

I guess the question here should be about the Knative Serving version running on the cluster. I would add this to the question and remove the `Output of git describe` here.

Reply:
sounds good, updated.